pkg/vectorize/cot/cot_test.go

```diff
@@ -0,0 +1,137 @@
+package cot
```

Comment:
the indentation format of this file doesn't consistent with the format above

Reply:
well, from my GitHub page, the indentation format doesn't seem right, I think you indented two spaces here in this file, but a tabulator in the other files above